### PR TITLE
Fixed and added some dutch translations

### PR DESCRIPTION
**Pull request by glvr182**
There were some spelling errors in the dutch translations so I fixed those.
Also I added some missing translations.

For the other dutchies: 
I'm not a language expert or something so it is very possible that I missed some things.

### CODE REVIEW

**Comment by mjarkk** [comment]
I'll see what i can do

File: pkg/i18n/dutch.go [hunk 1]
```diff
@@ -82,12 +82,18 @@ func addDutch(i18nObject *i18n.Bundle) error {
...
+			Other: "push",
+		}, &i18n.Message{
+			ID:    "pull",
+			Other: "pull",
 		}, &i18n.Message{
 			ID:    "addPatch",
 			Other: "verandering toevoegen",
 		}, &i18n.Message{
 			ID:    "edit",
-			Other: "verander",
```
Comment (mjarkk) [inline pkg/i18n/dutch.go]: I think **bewerken** is a better translation

File: pkg/i18n/dutch.go [hunk 2]
```diff
@@ -387,7 +402,7 @@ func addDutch(i18nObject *i18n.Bundle) error {
 			Other: `maak een pull-aanvraag`,
 		}, &i18n.Message{
 			ID:    "NoBranchOnRemote",
-			Other: `Deze tak bestaat niet op de afstandsbediening. U moet eerst op de afstandsbediening drukken.`,
+			Other: `Deze tak bestaat niet op de afstandsbediening. U moet het eerst naar de remote pushen.`,
```
Comment (mjarkk) [inline pkg/i18n/dutch.go]: I think this is a better translations:  
`Deze branch bestaat niet op de remote. U moet het eerst naar de remote pushen.`

File: pkg/i18n/dutch.go (hunk 1 above)
Comment (glvr182) [reply pkg/i18n/dutch.go]: agreed

File: pkg/i18n/dutch.go (hunk 2 above)
Comment (glvr182) [reply pkg/i18n/dutch.go]: Consider it done

**Comment by glvr182** [comment]
@jesseduffield done aswell

**Pull request merged**